src/Appwrite/Platform/Tasks/ScheduleExecutions.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Appwrite\Event\Func;
+use Cron\CronExpression;
+use Utopia\Database\Database;
+use Utopia\Pools\Group;
+
+class ScheduleExecutions extends ScheduleBase
+{
+    public const UPDATE_TIMER = 3; // seconds
+    public const ENQUEUE_TIMER = 4; // seconds
+
+    public static function getName(): string
+    {
+        return 'schedule-executions';
+    }
+
+    public static function getSupportedResource(): string
+    {
+        return 'execution';
+    }
+
+    protected function enqueueResources(Group $pools, Database $dbForConsole): void
+    {
+        foreach ($this->schedules as $schedule) {
+            if (!$schedule['active'] || CronExpression::isValidExpression($schedule['schedule'])) {
+                unset($this->schedules[$schedule['resourceId']]);
+                continue;
+            }
+
+            $now = new \DateTime();
+            $scheduledAt = new \DateTime($schedule['schedule']);
+
+            if ($scheduledAt > $now) {
+                continue;
+            }
+
+            \go(function () use ($schedule, $pools, $dbForConsole) {
```

Comment:
I think we don't need to do this in `go`. We do it for CRONs because we delay it some more with sleep. But here it's instant execution - we can do it without go.

Reply:
OK - removed.